rotkehlchen/api/rest.py

```diff
@@ -2273,8 +2273,8 @@ def _modify_manually_tracked_balances(  # pylint: disable=unused-argument, no-se
     def _modify_manually_tracked_balances(
             self,
             function: Union[
-                Callable[['DBHandler', list[ManuallyTrackedBalance]], None],
-                Callable[['DBHandler', list[int]], None],
+            Callable[['DBHandler', list[ManuallyTrackedBalance]], None],
```

Comment:
Okay so looking at output it has a bug for when the type of the argument breaks into multiple lines